galaxy/xcms_fillpeaks/abims_xcms_fillPeaks.xml

```diff
@@ -268,6 +268,10 @@ Output files
 Changelog/News
 --------------
 
+**Version 2.2.0-1.50.1 - 23/03/2017**
```

Comment:
I think so.
It can be see as:
- a warning "Be careful, we have change something behind even if you may not see the difference"
- an ad, "Hey, look at that, we are still working on our wrapper, and follow the XMCS developments"

We can also adopt the `USER VISIBLE CHANGES:` tag ?